test/lib/godmin/resources/resource_service/ordering_test.rb

```diff
@@ -18,7 +12,8 @@ def order(order_param)
           end
         end
 
-        @article_service = ArticleService.new(resource_class: resource_class)
+        @resources = @resources_class.new
+        @article_service = Fakes::ArticleService.new(resources: @resources)
       end
```

Comment:
Here we used a custom resource class. Not sure why so I removed it and use the `Fakes::ArticleService` instead, like in the other tests. I can revert it if we have a good reason for it :)